### PR TITLE
Add pre header to the email message and template

### PR DESCRIPTION
**Pull request by vichi99**

### CODE REVIEW

File: pymess/models/emails.py [hunk 1]
```diff
@@ -63,6 +63,7 @@ class EmailMessage(BaseMessage):
     sender = models.EmailField(verbose_name=_('sender'), blank=False, null=False)
     sender_name = models.CharField(verbose_name=_('sender name'), blank=True, null=True, max_length=250)
     subject = models.TextField(verbose_name=_('subject'), blank=False, null=False)
+    pre_header = models.CharField(verbose_name=_('pre header'), blank=True, null=True, max_length=100)
```
Comment (vichi99) [inline pymess/models/emails.py:66]: Only for saving to DB. For our purposes it will be sent through the template.

**Review by Formulka — APPROVED** [review]

File: pymess/models/emails.py (hunk 1 above)
Comment (rubickcz) [reply pymess/models/emails.py:66]: What's point of having it here, if there is no use for it.

Comment (vichi99) [reply pymess/models/emails.py:66]: To view in admin as a single attribute

File: pymess/locale/cs/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -416,95 +416,99 @@ msgstr "zakázaný objekt zprávy dialeru"
...
 
-#: models/emails.py:65 models/emails.py:160
+#: models/emails.py:65 models/emails.py:161
 msgid "subject"
 msgstr "předmět"
 
-#: models/emails.py:66
+#: models/emails.py:66 models/emails.py:162
+msgid "pre header"
+msgstr "hlavička"
```
Comment (vichi99) [inline pymess/locale/cs/LC_MESSAGES/django.po:433]: I've added a translation by `django-admin makemessages -a ` and `django-admin compilemessages`

**Review by rubickcz — APPROVED** [review]

**Pull request merged**